projects/packages/masterbar/src/admin-menu/class-base-admin-menu.php

```diff
@@ -547,6 +547,16 @@ public function add_dashboard_switcher() {
 		if ( empty( $menu_mappings[ $screen ] ) ) {
 			return;
 		}
+
+		/*
+		 * Don't show the switcher if we're enforcing the WP Admin views to participants of
+		 * the "Remove duplicate views" experiment.
+		 */
+		if ( defined( 'WPCOM_DUPLICATED_VIEW' ) && function_exists( 'wpcom_is_duplicate_views_experiment_enabled' ) ) {
```

Comment:
nit: Can we have this code in jetpack-mu-wpcom? We could de-register the action entirely. It would be easier to clean up after if we have them grouped.

Reply:
https://github.com/Automattic/jetpack/pull/40595/commits/496d2b93c7524810fa702d040d4df9a517fe8aea